### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by ruffle1986**

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+var lastChar = require('../lib')
```
Comment (IonicaBizau) [inline test/index.js]: Please use uppercase letters for dependency names. :smile:

File: test/index.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  , assert = require('assert')
+  ;
+
+describe('last-char', function () {
```
Comment (IonicaBizau) [inline test/index.js]: Also, double quotes instead of simple quotes.

**Comment by IonicaBizau** [comment]
Nice work! :sparkles:

File: test/index.js (hunk 2 above)
Comment (ruffle1986) [inline test/index.js]: Aw yeah, stupid me. Don't you want to use a static analysis tool like `eslint`?

Comment (IonicaBizau) [inline test/index.js]: That would be amazing! :sparkle: Or maybe is it possible to generate the code using some tool?

I'm not documented in this direction, but I'm interested to find solutions.

**Comment by IonicaBizau** [comment]
:ship: **`1.1.1`** :tada:

**Comment by IonicaBizau** [comment]
Thanks! :smile:

**Comment by ruffle1986** [comment]
Not at all :+1:

File: test/index.js (hunk 2 above)
Comment (ruffle1986) [inline test/index.js]: You should create your own shareable config for `eslint`. It'd make it easier to contribute and enforce devs to follow your style. Set the linting script as a pre-commit hook and voila. 

Have you heard about it?

You just set your eslint config and publish it as an npm module, and tell eslint to use it while linting.

http://eslint.org/docs/developer-guide/shareable-configs

Comment (IonicaBizau) [inline test/index.js]: Great! Thanks. 

I opened this issue: https://github.com/IonicaBizau/code-style/issues/4
We can continue the conversation there. :smile:

Comment (ruffle1986) [inline test/index.js]: Sure.

**Pull request merged**